File: src/data/hullmods/niko_MPC_cheapDeployment.java
package data.hullmods;

import com.fs.starfarer.api.Global;
import com.fs.starfarer.api.combat.*;
import com.fs.starfarer.api.impl.campaign.ids.Stats;

public class niko_MPC_cheapDeployment extends BaseHullMod {

    public static final float deploymentMult = 0f;

    @Override
    public void applyEffectsBeforeShipCreation(ShipAPI.HullSize hullSize, MutableShipStatsAPI stats, String id) {
        stats.getDynamic().getMod(Stats.DEPLOYMENT_POINTS_MOD).modifyMult(id, deploymentMult);
    }
}

File: src/data/hullmods/niko_MPC_cheapDeployment.kt
package data.hullmods

import com.fs.starfarer.api.combat.BaseHullMod
import com.fs.starfarer.api.combat.MutableShipStatsAPI
import com.fs.starfarer.api.combat.ShipAPI
import com.fs.starfarer.api.combat.ShipAPI.HullSize
import com.fs.starfarer.api.impl.campaign.ids.Stats

class niko_MPC_cheapDeployment : BaseHullMod() {

    companion object {
        const val deploymentMult = 0f
    }

    override fun applyEffectsBeforeShipCreation(hullSize: HullSize, stats: MutableShipStatsAPI, id: String) {
        stats.dynamic.getMod(Stats.DEPLOYMENT_POINTS_MOD).modifyMult(id, deploymentMult)
    }
}